Moderation_Matrix.md

```diff
@@ -0,0 +1,52 @@
+# Moderation Matrix
+
```

Comment:
A link to the Discord rules here could be useful as a reference (do they exist on a repo somewhere? Couldn't find them - I can add if needed)

Reply:
I think we should be using the newly updated rules that are available on our website. https://www.theodinproject.com/community_rules. Matter of fact, due to a recent conversation that I was having with maintainers, I think the final home of this matrix and these community rules needs to be one the same page. (This will reduce the places to update rules in the future and make it less information for users to read).